From 880c1dc81f012dcbddb6be5f625021a047434dd6 Mon Sep 17 00:00:00 2001
From: Arik Nemtsov <arik@wizery.com>
Date: Sun, 26 Jun 2011 12:54:52 +0300
Subject: [PATCH 15/32] hostapd: add android 11a hostapd.conf file and enable ctrl_interface

hostapd_cli can now operate through:

hostapd_cli -p/data/misc/wifi/sockets/
---
 android_hostapd.conf     |   78 ---------------------------------------------
 android_hostapd_11a.conf |   79 ++++++++++++++++++++++++++++++++++++++++++++++
 android_hostapd_11g.conf |   79 ++++++++++++++++++++++++++++++++++++++++++++++
 3 files changed, 158 insertions(+), 78 deletions(-)
 delete mode 100644 android_hostapd.conf
 create mode 100644 android_hostapd_11a.conf
 create mode 100644 android_hostapd_11g.conf

diff --git a/android_hostapd.conf b/android_hostapd.conf
deleted file mode 100644
index 117ae71..0000000
--- a/android_hostapd.conf
+++ /dev/null
@@ -1,78 +0,0 @@
-driver=nl80211
-logger_syslog=-1
-logger_syslog_level=2
-logger_stdout=-1
-logger_stdout_level=2
-dump_file=/data/misc/wifi/hostapd.dump
-hw_mode=g
-channel=11
-beacon_int=100
-dtim_period=2
-supported_rates=60 120 240 360 480 540
-basic_rates=60 120 240
-preamble=1
-ignore_broadcast_ssid=0
-own_ip_addr=127.0.0.1
-wpa_group_rekey=0
-wpa_gmk_rekey=0
-wpa_ptk_rekey=0
-interface=wlan1
-ssid=AndroidAP
-ieee80211d=1
-country_code=US
-max_num_sta=5
-macaddr_acl=0
-auth_algs=3
-tx_queue_data3_aifs=7
-tx_queue_data3_cwmin=15
-tx_queue_data3_cwmax=1023
-tx_queue_data3_burst=0
-tx_queue_data2_aifs=3
-tx_queue_data2_cwmin=15
-tx_queue_data2_cwmax=63
-tx_queue_data2_burst=0
-tx_queue_data1_aifs=1
-tx_queue_data1_cwmin=7
-tx_queue_data1_cwmax=15
-tx_queue_data1_burst=3.0
-tx_queue_data0_aifs=1
-tx_queue_data0_cwmin=3
-tx_queue_data0_cwmax=7
-tx_queue_data0_burst=1.5
-wme_enabled=1
-wme_ac_bk_cwmin=4
-wme_ac_bk_cwmax=10
-wme_ac_bk_aifs=7
-wme_ac_bk_txop_limit=0
-wme_ac_bk_acm=0
-wme_ac_be_aifs=3
-wme_ac_be_cwmin=4
-wme_ac_be_cwmax=10
-wme_ac_be_txop_limit=0
-wme_ac_be_acm=0
-wme_ac_vi_aifs=2
-wme_ac_vi_cwmin=3
-wme_ac_vi_cwmax=4
-wme_ac_vi_txop_limit=94
-wme_ac_vi_acm=0
-wme_ac_vo_aifs=2
-wme_ac_vo_cwmin=2
-wme_ac_vo_cwmax=3
-wme_ac_vo_txop_limit=47
-wme_ac_vo_acm=0
-uapsd_advertisement_enabled=1
-wep_rekey_period=0
-own_ip_addr=127.0.0.1
-wpa_group_rekey=0
-wpa_gmk_rekey=0
-wpa_ptk_rekey=0
-ap_table_max_size=255
-ap_table_expiration_time=60
-wps_state=2
-ap_setup_locked=1
-uuid=12345678-9abc-def0-1234-56789abcdef0
-eap_server=1
-disassoc_low_ack=1
-ap_max_inactivity=10000
-ieee80211n=1
-ht_capab=[SHORT-GI-20][GF]
diff --git a/android_hostapd_11a.conf b/android_hostapd_11a.conf
new file mode 100644
index 0000000..c9b851f
--- /dev/null
+++ b/android_hostapd_11a.conf
@@ -0,0 +1,79 @@
+driver=nl80211
+logger_syslog=-1
+logger_syslog_level=2
+logger_stdout=-1
+logger_stdout_level=2
+dump_file=/data/misc/wifi/hostapd.dump
+ctrl_interface=/data/misc/wifi/sockets
+hw_mode=a
+channel=36
+beacon_int=100
+dtim_period=2
+supported_rates=60 120 240 360 480 540
+basic_rates=60 120 240
+preamble=1
+ignore_broadcast_ssid=0
+own_ip_addr=127.0.0.1
+wpa_group_rekey=0
+wpa_gmk_rekey=0
+wpa_ptk_rekey=0
+interface=wlan1
+ssid=AndroidAP
+ieee80211d=1
+country_code=US
+max_num_sta=5
+macaddr_acl=0
+auth_algs=3
+tx_queue_data3_aifs=7
+tx_queue_data3_cwmin=15
+tx_queue_data3_cwmax=1023
+tx_queue_data3_burst=0
+tx_queue_data2_aifs=3
+tx_queue_data2_cwmin=15
+tx_queue_data2_cwmax=63
+tx_queue_data2_burst=0
+tx_queue_data1_aifs=1
+tx_queue_data1_cwmin=7
+tx_queue_data1_cwmax=15
+tx_queue_data1_burst=3.0
+tx_queue_data0_aifs=1
+tx_queue_data0_cwmin=3
+tx_queue_data0_cwmax=7
+tx_queue_data0_burst=1.5
+wme_enabled=1
+wme_ac_bk_cwmin=4
+wme_ac_bk_cwmax=10
+wme_ac_bk_aifs=7
+wme_ac_bk_txop_limit=0
+wme_ac_bk_acm=0
+wme_ac_be_aifs=3
+wme_ac_be_cwmin=4
+wme_ac_be_cwmax=10
+wme_ac_be_txop_limit=0
+wme_ac_be_acm=0
+wme_ac_vi_aifs=2
+wme_ac_vi_cwmin=3
+wme_ac_vi_cwmax=4
+wme_ac_vi_txop_limit=94
+wme_ac_vi_acm=0
+wme_ac_vo_aifs=2
+wme_ac_vo_cwmin=2
+wme_ac_vo_cwmax=3
+wme_ac_vo_txop_limit=47
+wme_ac_vo_acm=0
+uapsd_advertisement_enabled=1
+wep_rekey_period=0
+own_ip_addr=127.0.0.1
+wpa_group_rekey=0
+wpa_gmk_rekey=0
+wpa_ptk_rekey=0
+ap_table_max_size=255
+ap_table_expiration_time=60
+wps_state=2
+ap_setup_locked=1
+uuid=12345678-9abc-def0-1234-56789abcdef0
+eap_server=1
+disassoc_low_ack=1
+ap_max_inactivity=10000
+ieee80211n=1
+ht_capab=[SHORT-GI-20][GF]
diff --git a/android_hostapd_11g.conf b/android_hostapd_11g.conf
new file mode 100644
index 0000000..728ca20
--- /dev/null
+++ b/android_hostapd_11g.conf
@@ -0,0 +1,79 @@
+driver=nl80211
+logger_syslog=-1
+logger_syslog_level=2
+logger_stdout=-1
+logger_stdout_level=2
+dump_file=/data/misc/wifi/hostapd.dump
+ctrl_interface=/data/misc/wifi/sockets
+hw_mode=g
+channel=11
+beacon_int=100
+dtim_period=2
+supported_rates=60 120 240 360 480 540
+basic_rates=60 120 240
+preamble=1
+ignore_broadcast_ssid=0
+own_ip_addr=127.0.0.1
+wpa_group_rekey=0
+wpa_gmk_rekey=0
+wpa_ptk_rekey=0
+interface=wlan1
+ssid=AndroidAP
+ieee80211d=1
+country_code=US
+max_num_sta=5
+macaddr_acl=0
+auth_algs=3
+tx_queue_data3_aifs=7
+tx_queue_data3_cwmin=15
+tx_queue_data3_cwmax=1023
+tx_queue_data3_burst=0
+tx_queue_data2_aifs=3
+tx_queue_data2_cwmin=15
+tx_queue_data2_cwmax=63
+tx_queue_data2_burst=0
+tx_queue_data1_aifs=1
+tx_queue_data1_cwmin=7
+tx_queue_data1_cwmax=15
+tx_queue_data1_burst=3.0
+tx_queue_data0_aifs=1
+tx_queue_data0_cwmin=3
+tx_queue_data0_cwmax=7
+tx_queue_data0_burst=1.5
+wme_enabled=1
+wme_ac_bk_cwmin=4
+wme_ac_bk_cwmax=10
+wme_ac_bk_aifs=7
+wme_ac_bk_txop_limit=0
+wme_ac_bk_acm=0
+wme_ac_be_aifs=3
+wme_ac_be_cwmin=4
+wme_ac_be_cwmax=10
+wme_ac_be_txop_limit=0
+wme_ac_be_acm=0
+wme_ac_vi_aifs=2
+wme_ac_vi_cwmin=3
+wme_ac_vi_cwmax=4
+wme_ac_vi_txop_limit=94
+wme_ac_vi_acm=0
+wme_ac_vo_aifs=2
+wme_ac_vo_cwmin=2
+wme_ac_vo_cwmax=3
+wme_ac_vo_txop_limit=47
+wme_ac_vo_acm=0
+uapsd_advertisement_enabled=1
+wep_rekey_period=0
+own_ip_addr=127.0.0.1
+wpa_group_rekey=0
+wpa_gmk_rekey=0
+wpa_ptk_rekey=0
+ap_table_max_size=255
+ap_table_expiration_time=60
+wps_state=2
+ap_setup_locked=1
+uuid=12345678-9abc-def0-1234-56789abcdef0
+eap_server=1
+disassoc_low_ack=1
+ap_max_inactivity=10000
+ieee80211n=1
+ht_capab=[SHORT-GI-20][GF]
-- 
1.7.0.4

